src/index.js

```diff
@@ -200,6 +192,62 @@ class FloodSub extends EventEmitter {
   }
 
   /**
+   * Mounts the floodsub protocol onto the libp2p node and sends our
+   * subscriptions to every peer conneceted
+   *
+   * @param {Function} callback
+   * @returns {undefined}
+   *
+   */
+  start (callback) {
```

Comment:
remembering the discussions for the better js-ipfs start and stop interface, could we make this callback just bind to a start event and similarly for stop?